FlowCrypt/Functionality/Services/Organisational Rules Service/OrganisationalRulesService.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  OrganisationalRulesService.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 18.06.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
```

Comment:
Let's rename this to `Client Configuration Service/ClientConfigurationService.swift`.